src/Test.UnitTests.Sarif/PropertiesDictionaryTests.cs

```diff
@@ -192,6 +227,10 @@ private PropertiesDictionary RoundTripThroughJson(PropertiesDictionary propertie
             new PerLanguageOption<StringSet>(
                 FEATURE, nameof(StringSetProperty), defaultValue: () => { return STRINGSET_DEFAULT; });
 
+        public static PerLanguageOption<StringSet> GetStringSetProperty(int i) =>
```

Comment:
>GetStringSetProperty [](http://example.com/codeflow?start=51&length=20)

I would change to StringGetProperty

Reply:
StringSet is a actually class here, let me know 
how about change to "GenerateStringSetProperty"
the Get word may be confusing

Reply:
'StringSet' is correct. :) I can completely understand why 'GenerateStringGetProperty' seems right, though! :)